grafana-operator/grafana-operator
# correct spec.deployment.annotations

add the spec.deployment.annotations in grafana CR to the grafana deployment

Signed-off-by: Moulick Aggarwal <moulickaggarwal@gmail.com>


## Description
The spec.deployment.annotations are now also added to the grafana deployments in addition to pod template

## Relevant issues/tickets
Closes https://github.com/grafana-operator/grafana-operator/issues/697

## Type of change




## Checklist


## Verification steps

1. Create a grafana with any annotation to spec.deployment.annotations
2. Check if the annotation is applied to the resulting grafana deployment
